flow/flow.h

```diff
@@ -687,6 +687,11 @@ class Future
 	{
 		sav->send(presentValue);
 	}
+	Future(T&& presentValue)
+		: sav(new SAV<T>(1, 0))
+	{
+		sav->send(std::forward<T>(presentValue));
```

Comment:
Nice find!

This isn't actually a [forwarding reference](https://en.cppreference.com/w/cpp/language/reference#Forwarding_references) - it's an rvalue reference. I think std::move would be clearer